R/parameter-range.R

```diff
@@ -52,19 +55,35 @@ ParameterRange <- R6::R6Class(
     #' Print the object to the console
     #' @param ... Rest arguments.
     print = function(...) {
-      private$.printClass()
-      private$.printLine("Min", self$min)
-      private$.printLine("Max", self$max)
-      private$.printLine("Unit", self$unit)
-      invisible(self)
+      ospsuite.utils::ospPrintClass(self)
+      ospsuite.utils::ospPrintItems(
+        list(
+          "Min" = self$min,
+          "Max" = self$max,
+          "Unit" = self$unit
+        ),
+        print_empty = TRUE
+      )
     },
     #' @description
-    #' Print the the parameter in one line
+    #' Print  the parameter in one line
     #' @param caption Caption to display before the value of the parameter
     printValue = function(caption) {
+      lifecycle::deprecate_warn(
```

Comment:
Added a function `getPrintValue` that formats the string for output. This was it can be used in print methods of other classes.